src/autopas/LogicHandler.h

```diff
@@ -317,7 +319,7 @@ class LogicHandler {
     }
     Particle particleCopy = p;
     particleCopy.setOwnershipState(OwnershipState::owned);
-    if (not _neighborListsAreValid.load(std::memory_order_relaxed)) {
+    if (forceToContainer) {
       // Container has to (about to) be invalid to be able to add Particles!
```

Comment:
Update comment

Reply:
I simply deleted it. I think the new code is self-explanatory.